extensions/data-workspace/src/common/constants.ts

```diff
@@ -23,6 +23,7 @@ export const gitCloneError = localize('gitCloneError', "Error during git clone.
 export const openedProjectsUndefinedAfterRefresh = localize('openedProjectsUndefinedAfterRefresh', "List of opened projects should not be undefined after refresh from disk.");
 export const dragAndDropNotSupported = localize('dragAndDropNotSupported', "This project type does not support drag and drop.");
 export const onlyMovingOneFileIsSupported = localize('onlyMovingOneFileIsSupported', "Only moving one file at a time is supported.");
+export const noProjectProvidingExtensionsInstalled = localize('noProjectProvidingExtensionsInstalled', "No database project extensions are installed. Please install a database project extension to use this feature.");
```

Comment:
Is "database project extensions" completely correct, or should this be "project extensions"?

Reply:
The title of the view is "Database Projects" so I think it's ok. Otherwise people might try looking for a C# project extension and expect it to work here